Limit width of code-blocks to prevent wraparound of TOC (#1224)
redpanda-data/documentation: modules/get-started/pages/release-notes/redpanda.adoc

* xref:redpanda-cloud:get-started:whats-new-cloud.adoc[]
* xref:redpanda-cloud:get-started:cloud-overview.adoc#redpanda-cloud-vs-self-managed-feature-compatibility[Redpanda Cloud vs Self-Managed feature compatibility]

== Schema Registry Authorization

> You can now use the Enterprise-licensed feature xref:manage:schema-reg-authorization.adoc[Schema Registry Authorization] to control access to Schema Registry subjects and operations using either `rpk` or the xref:api:ROOT:schema-registry-api.adoc#get-/security/acls[Redpanda Schema Registry API] endpoints. Schema Registry Authorization offers more granular control over who can do what with your Redpanda Schema Registry data. ACLs used for Schema Registry access also support RBAC roles.

Reviewer: The link has the note that this is an Enterprise feature, I don't think we need to mention it in the release notes.
```suggestion
xref:manage:schema-reg-authorization.adoc[Schema Registry Authorization] is now available to control access to Schema Registry subjects and operations using either `rpk` or the xref:api:ROOT:schema-registry-api.adoc#get-/security/acls[Redpanda Schema Registry API] endpoints. Schema Registry Authorization offers more granular control over who can do what with your Redpanda Schema Registry data. ACLs used for Schema Registry access also support RBAC roles.
```

Reviewer 2: Despite the filename, this is for the What's New in 25.2 topic @treevon 
It's super important that whatever the context--whether you are browsing what the new features are, or you are using the feature--that users understand that it requires an Enterprise license. Going to keep this here.

> You can now use the Enterprise-licensed feature xref:manage:schema-reg-authorization.adoc[Schema Registry Authorization] to control access to Schema Registry subjects and operations using either `rpk` or the xref:api:ROOT:schema-registry-api.adoc#get-/security/acls[Redpanda Schema Registry API] endpoints. Schema Registry Authorization offers more granular control over who can do what with your Redpanda Schema Registry data. ACLs used for Schema Registry access also support RBAC roles.

Reviewer 3: ```suggestion
xref:manage:schema-reg-authorization.adoc[Schema Registry authorization] is now available, enabling fine-grained restriction of operations to Schema Registry resources by user or role. Management of ACLs is available through rpk or the xref:api:ROOT:schema-registry-api.adoc#get-/security/acls[Schema Registry API].

See also: xref:manage:security/authentication.adoc#sasl[Configure Authentication] and xref:manage:security/authorization/acl.adoc[]

== Schema Registry Authorization

> You can now use the Enterprise-licensed feature xref:manage:schema-reg-authorization.adoc[Schema Registry Authorization] to control access to Schema Registry subjects and operations using either `rpk` or the xref:api:ROOT:schema-registry-api.adoc#get-/security/acls[Redpanda Schema Registry API] endpoints. Schema Registry Authorization offers more granular control over who can do what with your Redpanda Schema Registry data. ACLs used for Schema Registry access also support RBAC roles.

Reviewer 4: @Feediver1 'resources' is a better term than 'data' here (like you used above in the licensing section).  'data' confuses this a bit with messages in a topic
